app/controllers/api/v1/orders_controller.rb

```diff
@@ -41,6 +41,6 @@ def destroy
   private
 
   def order_params
-    params.require(:order).permit(:phone, :school_name, :school_address, :school_year, :kit_id, :comments)
+    params.require(:order).permit(:phone, :school_year, :kit_id, :comments)
```

Comment:
Oh. Here's something we need to decide on. 

Where can a `User` set their `Address`?

1. `Order` could still send an `address` as [nested attributes](https://api.rubyonrails.org/classes/ActionController/Parameters.html#method-i-permit)
   1. The UI pre-fills if they have an address saved, and allows them to update with the act of ordering, either as a popup modal or directly in the form
   2. The controller handles updating the User's address
3. The `Order` sends the `user_id` as an attribute
   1. The address is displayed to the user on the order form, but editing it makes a request to an `AddressesController` 
4. Some third option???

@mhope21 @olesiamironenko @trca831 @dewi-anggraini @bbcc33 Interested to hear opinions.

Reply:
option 1 looks appealing to me

Reply:
I agree with @olesiamironenko, Option #1 looks good. Is it possible to do it with one ticket or would it need a few tickets to accomplish it?

Reply:
I think 2 makes more sense to me. We have an address model, so an address controller makes sense. The ui can show existing address gotten from users relationship to order. Then, maybe there is an edit icon next to the address section that opens a modal to make changes. This calls to address controller to update the address. Maybe I am misunderstanding something though. I am also curious about the organization address. I guess we are saying that a user can choose for their associated address to match the organization's, but they don't have to. Regardless whichever address they associate with themselves is the one the order goes to?

Reply:
Could the user also set their address on their profile?

Reply:
Ok. Looks like option 1 wins. 

`Order` should be able to send nested attributes for `address` OR an `address_id` if the user already has an address saved to their account.

We can add a follow ticket for this PR to add that to the necessary APIs. We also want to make sure that `User` gets serialized with the address, including the id so that this information is available to the frontend.